CMR/python/cmr/auth/token.py

```diff
@@ -115,7 +121,10 @@ def token_manager(config: dict = None):
 
 def token(token_lambdas = None, config: dict = None):
     """
-    Recursively calls lambdas till a token is found
+    Recursively calls lambdas till a token is found. These lamdba functions
```

Comment:
Change the word Recursively to iterate through the list until a token is found. or something like that.